spark-bigquery-dsv2/spark-3.5-bigquery/src/test/java/com/google/cloud/spark/bigquery/integration/Spark35ReadByFormatIntegrationTest.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.spark.bigquery.integration;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import com.google.cloud.bigquery.*;
+import java.time.LocalDateTime;
+import org.apache.spark.sql.Dataset;
+import org.apache.spark.sql.Row;
+import org.apache.spark.sql.types.DataTypes;
+import org.apache.spark.sql.types.StructType;
+import org.junit.Test;
+
+public class Spark35ReadByFormatIntegrationTest extends ReadByFormatIntegrationTestBase {
+
+  public Spark35ReadByFormatIntegrationTest() {
+    super("ARROW", /* userProvidedSchemaAllowed */ false);
+  }
+
+  // tests are from the super-class
+  @Test
+  public void testTimestampNTZReadFromBigQuery() {
```

Comment:
Since this is copied from the Spark 3.4 test, can you either (1) refactor the original method to be static and call it from here; or (2) inherit from `Spark35ReadByFormatIntegrationTest ` ?